src/features/dsComponents/DatasetComponent.tsx

```diff
@@ -16,11 +16,14 @@ import IconLink from '../../chrome/IconLink'
 export interface DatasetComponentProps {
   dataset: Dataset
   componentName: ComponentName
+  // preview will cause the body component to render only what is in dataset and not fetch more data
+  preview?: boolean
```

Comment:
not a massive fan of this "preview" prop name, but given that it's already landed in the `Body` component, let's stick with it. I'd prefer `noFetch`, which would make the behavior it's affecting more accurate